src/hooks/useNotificationTypes.tsx

```diff
@@ -653,6 +655,42 @@ export const notificationTypes: NotificationTypeConfig[] = [
       };
     },
   },
+  {
+    type: NotificationType.FeedbackRequest,
+    useTrigger: ({ trigger }) => {
+      const { dydxAddress } = useAccounts();
+      const { getInTouch } = useURLConfigs();
+      const feedbackRequestWalletAddresses = useEnvConfig('feedbackRequestWalletAddresses');
+      const stringGetter = useStringGetter();
+
+      useEffect(() => {
+        if (dydxAddress && feedbackRequestWalletAddresses.includes(dydxAddress) && getInTouch) {
+          trigger(FeedbackRequestNotificationIds.Top100UserSupport, {
```

Comment:
dismissal and saving the dismissed state and everything is handled elsewhere right? Just wanted to double check this won't be popping up constantly.

Reply:
yesssir